hx_requests/views.py

```diff
@@ -21,19 +21,21 @@ class HtmxViewMixin(View):
 
     hx_requests: Dict = []
 
-    def get(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
-        if is_htmx_request(request):
-            hx_request = self._setup_hx_request(request, *args, **kwargs)
-            kwargs = self.get_extra_kwargs(request)
-            return hx_request.get(request, *args, **kwargs)
-        return super().get(request, *args, **kwargs)
-
-    def post(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
-        if is_htmx_request(request):
-            hx_request = self._setup_hx_request(request, *args, **kwargs)
-            kwargs = self.get_extra_kwargs(request)
-            return hx_request.post(request, *args, **kwargs)
-        return super().post(request, *args, **kwargs)
+    def dispatch(self, request, *args, **kwargs):
+        # Try to dispatch to the right method; if a method doesn't exist,
+        # defer to the error handler. Also defer to the error handler if the
+        # request method isn't on the approved list.
+        if request.method.lower() in self.http_method_names:
```

Comment:
Can you please a line to the comments how the handler is conditional on hx request or not, if its an htmx request then the handler is the hx_request otherwise its the view